ui/lib/apps/SlowQuery/pages/Detail/DetailTabBasic.tsx

```diff
@@ -2,13 +2,15 @@ import React from 'react'
 import { SlowquerySlowQuery } from '@lib/client'
 import { CardTableV2, DateTime } from '@lib/components'
 import { getValueFormat } from '@baurine/grafana-value-formats'
-import * as useColumn from '@lib/utils/useColumn'
+import { valueColumns } from '@lib/utils/table-columns'
```

Comment:
prefer `tableColumns`

Reply:
because there is another method called `timeValueColumns()`...